libs/dh/market-participant/actors/feature-delegation/src/lib/stop/dh-delegation-stop-modal.component.ts

```diff
@@ -65,11 +75,23 @@ import { DhDelegation } from '../dh-delegations';
       [formGroup]="stopDelegationForm"
       (ngSubmit)="stopSelectedDelegations()"
     >
-      <vater-stack align="flex-start">
-        <watt-datepicker-v2
-          [label]="t('stopDate')"
-          [formControl]="stopDelegationForm.controls.stopDate"
-        />
+      <vater-stack align="flex-start" gap="m">
+        <watt-radio
+          group="stopDate"
+          [formControl]="stopDelegationForm.controls.selectedOptions"
+          value="stopNow"
+          >{{ t('stopNow') }}</watt-radio
+        >
+        <vater-stack direction="row" gap="m">
+          <watt-radio
+            group="stopDate"
+            [formControl]="stopDelegationForm.controls.selectedOptions"
+            value="stopOnDate"
+          >
+            {{ t('stopDate') }}
+          </watt-radio>
+          <watt-datepicker-v2 [min]="date" [formControl]="stopDelegationForm.controls.stopDate" />
```

Comment:
issue: looks a bit funny with the lonely star above the input. 

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/e53d87d5-f869-4858-b286-f4ccc3777c81)

Reply:
Its because its required, but have no label. The label is kind a the radio button. Do have a good idea how to solve this?